public/app/features/explore/NodeGraph/NodeGraphContainer.tsx

```diff
@@ -81,12 +80,12 @@ export function UnconnectedNodeGraphContainer(props: Props) {
     ) : null;
 
   return (
-    <Collapse
-      label={<span>Node graph{countWarning} </span>}
-      collapsible={withTraceView}
+    <PanelChrome
+      title={`Node graph${countWarning}`}
```

Comment:
This now shows `null` in the title when using test data:

<img width="500" alt="Screenshot 2023-08-01 at 14 56 54" src="https://github.com/grafana/grafana/assets/745532/fe3781e8-fc9d-4dc6-9ba3-062904d3049b">

Reply:
Since `title` accepts only a string countWarning wouldn't display correctly if it had a value  because it's wrapped in a span. I've fixed this by passing `countWarning` to  PanelChrome's `titleItems` prop. I think it looks nice and it's good to have this message separate from the title. @grafana/observability-traces-and-profiling, any thoughts on this?

Reply:
Nice! I think just to make it consistent with what's currently in main we can align the message to the middle with:

```typescript
const getStyles = (theme: GrafanaTheme2) => ({
  warningText: css`
    display: flex;
    align-items: center;
    label: warningText;
    font-size: ${theme.typography.bodySmall.fontSize};
    color: ${theme.colors.text.secondary};
  `,
});
```

<img width="357" alt="Screenshot 2023-08-02 at 08 53 41" src="https://github.com/grafana/grafana/assets/745532/a178490e-43ee-4eab-adb2-1304195ef4f9">